client/account-recovery/reset-password/reset-password-confirm-form/style.scss

```diff
@@ -0,0 +1,50 @@
+.reset-password-confirm-form__title {
+	font-size: 20px;
+	font-weight: 600;
+	margin-bottom: 16px;
+}
+
+.form-button.reset-password-confirm-form__button {
```

Comment:
Just like @omarjackman said, I have to use a more specific selector like this here to override the properties I wanted. However, to be honest, I just *found* that it works, instead of *knowing* why I needed to be so specific in this case.